estilo/syntax/base.yml

```diff
@@ -48,9 +48,9 @@ StatusLine: 'foreground background r'
 StatusLineTerm: '@StatusLine'
 StatusLineTermNC: 'pureBlack darkPurp r'
 StatusLineNC: '@StatusLineTermNC'
-TabLine: 'pureBlack grey'
-TabLineFill: 'grey pureBlack'
-TabLineSel: 'grey2 background'
+TabLine: 'foreground background'
+TabLineFill: 'darkPurp2 background'
+TabLineSel: 'purple background'
```

Comment:
There isn't anything in Estilo for Airline tabline, but in `syntax/base.yml`, these three lines seem to be incorrect so I matched it according to the new changes, but to be 100% transparent, I have no idea how to use Estilo.

Reply:
I’ll test it no worries. I’m gonna deprecate it soon as it’s causing me more pain then anything else these days.